src/dawn/CodeGen/Cuda/MSCodeGen.cpp

```diff
@@ -255,7 +300,12 @@ void MSCodeGen::generateFillKCaches(MemberFunction& cudaKernel, const iir::Inter
 
     DAWN_ASSERT(cache.getInterval().is_initialized());
     const auto cacheInterval = *(cache.getInterval());
-    auto vertExtent = cacheProperties_.getKCacheVertExtent(accessID);
+    auto extents = ms_->computeExtents(accessID, interval);
```

Comment:
what does getKCacheVertExtent do now? It's still used in two places (declaration and slide)

Reply:
thanks for clarification